cc/client/client_test.cc

```diff
@@ -48,17 +47,17 @@ constexpr uint8_t kTestSessionSize = 8;
 // TODO(#3641): Send test remote attestation report to the client and add corresponding tests.
 class TestTransport : public TransportWrapper {
  public:
-  TestTransport() : key_pair_(*KeyPair::Generate()) {}
+  TestTransport() : encryption_key_provider_(*EncryptionKeyProvider::Create()) {}
```

Comment:
This is undefined behaviour. You're dereferencing a `StatusOr` with `operator*()` without first checking if it's `ok()`.

https://github.com/abseil/abseil-cpp/blob/cffc9ef2b1174dea2b78cbf5efeeaea1606ad25b/absl/status/statusor.h#L534

Reply:
Done